tidy3d/components/medium.py

```diff
@@ -181,6 +182,13 @@ def _validate_nonlinear_spec(cls, val):
             f"currently supported for medium class {cls}."
         )
 
+    heat_spec: Optional[HeatSpecType] = pd.Field(
+        None,
+        title="Heat Specification",
+        description="Specification of the medium heat properties.",
```

Comment:
maybe we can add more description here. For example something like
```
If a heat_spec is added to a medium, it will be registered if the corresponding ``Structure`` is added to a ``HeatSimulation``.  <insert something about how to run heat sim and convert to optical sim>.  Otherwise, the ``heat_spec`` will not affect the running of an optical ``Simulation``.
```
Basically, just give a curious user looking at the medium API the opportunity to learn more about the heat solver.